codex-ui/src/vue/components/picture/picture.vue

```diff
@@ -397,6 +397,20 @@
       </clipPath>
     </defs>
   </svg>
+  <svg
+    v-if="name === 'NotexLogo'"
+    style="fill: var(--base--text);"
```

Comment:
it's better to use `fill: currentColor` inside the svg to allow manipulating color from outside